WalletWasabi.Gui/Converters/StatusBarStatusStringConverter.cs

```diff
@@ -36,7 +36,7 @@ public static string Convert(StatusBarStatus status)
 				case StatusBarStatus.BuildingTransaction: return "Building transaction...";
 				case StatusBarStatus.SigningTransaction: return "Signing transaction...";
 				case StatusBarStatus.BroadcastingTransaction: return "Broadcasting transaction...";
-				case StatusBarStatus.DequeuingSelectedCoins: return "Dequeueing selected coins...";
+				case StatusBarStatus.DequeuingSelectedCoins: return "Dequeuing selected coins...";
```

Comment:
Here fix still missing.

Reply:
@MaxHillebrand, thanks for the review. Found more and fixed the spelling. Could you please have a look again? Thanks.

Reply:
This change is good. `dequeuing` is shorter than `dequeueing`